src/modules/media/media.types.ts

```diff
@@ -12,6 +12,10 @@ export enum MediaFormat {
 	AUDIO = 'audio',
 }
 
+export type SeoInfo = {
+	title: string;
+};
```

Comment:
Use and extend the full type

```suggestion
export type MediaSeo = Pick<Media, 'name'>
```